SDK_reference.md

```diff
@@ -1,242 +1,3 @@
-# Client SDK reference:
```

Comment:
Hi

Reply:
Hello how are you doing? And I hope work is going well, I will be waiting

Reply:
> Hi

What are you waiting for, my account is not safe so neglect any massage or task or change for now. Thank you all for understanding and cooperation, do not accept any changes so work with our first agreement